src/test/java/org/mapdb/io/DataIOTest.java

```diff
@@ -1,19 +1,50 @@
 package org.mapdb.io;
 
+import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
 import org.mapdb.DBException;
-
-
-import java.io.*;
```

Comment:
We wouldn't normally ADD a wildcard import but I don't think we should be removing one that's already there (as it's the project-owner's choice)

Reply:
Yeah that's why I left it as a wildcard, was not sure